CHANGELOG.md

```diff
@@ -1,6 +1,23 @@
+## Version 0.28.0 (2022-09-07)
+
+[PR 314](https://github.com/fluencelabs/aquavm/pull/314):  
+The length functor was introduced
+
+[PR 307](https://github.com/fluencelabs/aquavm/pull/307):  
+bug with iteration over empty canonicalized stream was fixed
+
+[PR 305](https://github.com/fluencelabs/aquavm/pull/305):  
+bug with absence of join behaviour for canonicalized streams in fold instructions was fixed
+
+[PR 303](https://github.com/fluencelabs/aquavm/pull/303):  
+unsafe code for `LambdaAST` creation was removed
+
+[PR 301](https://github.com/fluencelabs/aquavm/pull/301):  
+bug with adding just executed values to a restricted stream was fixed
+
 ## Version 0.27.0 (2022-08-23)
 
-[PR 292](https://github.com/fluencelabs/aquavm/pull/292):
+[PR 292](https://github.com/fluencelabs/aquavm/pull/292):  
```

Comment:
Are these spaces (here and below) really needed?

Reply:
Yep, in `.md` file they lead to a newline.